z7/remove_repeating.hpp

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+void removeDuplicates(std::vector<std::string>&);
```

Comment:
this function should take std::string as an argument, not std::vector<std::string>